flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/FunctionCatalog.java

```diff
@@ -226,7 +235,8 @@ private void registerFunction(String name, FunctionDefinition functionDefinition
 		userFunctions.put(normalizeName(name), functionDefinition);
 	}
 
-	private String normalizeName(String name) {
+	@VisibleForTesting
+	protected static String normalizeName(String name) {
```

Comment:
It seems that the scope can be  "package" level. "protected" doesn't seem applicable since this is a static method.